vendor/licenses/grammar/language-faust.txt

```diff
@@ -0,0 +1,52 @@
+---
+type: grammar
+name: language-faust
+version: 93a620e7db713a9858c123e5be5e880ca53856c9
+license: other
```

Comment:
@lildude I'm not sure I remember how we're supposed to whitelist licenses now? This one looks okay, it's just duplicated with the original repository's license. Would just changing the above line to `license: MIT` be okay?

Reply:
Yes, manually changing this to mit should do the trick.

Reply:
Hi, I have edited the license as suggested.